docs/source/audio/configs.rst

```diff
@@ -0,0 +1,266 @@
+
+NeMo Dataset Configuration
+--------------------------
+
+Training, validation, and test parameters are specified using the ``model.train_ds``, ``model.validation_ds``, and ``model.test_ds`` sections in the configuration file, respectively.
+Depending on the task, there may be arguments specifying the sample rate or duration of the loaded audio examples. Some fields can be left out and specified via the command-line at runtime.
+Refer to the `Dataset Processing Classes <./api.html#Datasets>`__ section of the API for a list of datasets classes and their respective parameters.
+An example train, validation and test datasets can be configured as follows:
+
+.. code-block:: yaml
+
+  model:
+    sample_rate: 16000
+    skip_nan_grad: false
+
+    train_ds:
+      manifest_filepath: ???
+      input_key: audio_filepath # key of the input signal path in the manifest
+      target_key: target_filepath # key of the target signal path in the manifest
+      target_channel_selector: 0 # target signal is the first channel from files in target_key
+      audio_duration: 4.0 # in seconds, audio segment duration for training
+      random_offset: true # if the file is longer than audio_duration, use random offset to select a subsegment
+      min_duration: ${model.train_ds.audio_duration}
+      batch_size: 64 # batch size may be increased based on the available memory
+      shuffle: true
+      num_workers: 8
+      pin_memory: true
+
+    validation_ds:
+      manifest_filepath: ???
+      input_key: audio_filepath # key of the input signal path in the manifest
+      target_key: target_filepath
+      target_channel_selector: 0 # target signal is the first channel from files in target_key
+      batch_size: 64 # batch size may be increased based on the available memory
+      shuffle: false
+      num_workers: 4
+      pin_memory: true
+
+    test_ds:
+      manifest_filepath: ???
+      input_key: audio_filepath # key of the input signal path in the manifest
+      target_key: target_filepath # key of the target signal path in the manifest
+      target_channel_selector: 0 # target signal is the first channel from files in target_key
+      batch_size: 1 # batch size may be increased based on the available memory
+      shuffle: false
+      num_workers: 4
+      pin_memory: true
+
+More information about online augmentation can found in the `example configuration <https://github.com/NVIDIA/NeMo/blob/main/examples/audio/conf/masking.yaml>`_
+
+
+Lhotse Dataset Configuration
+----------------------------
+
+Lhotse CutSet
+~~~~~~~~~~~~~
+
+An example train dataset in Lhotse CutSet format can be configured as follows:
+
+.. code-block:: yaml
+
+  train_ds:
+    use_lhotse: true # enable Lhotse data loader
+    cuts_path: ??? # path to Lhotse cuts manifest with speech signals for augmentation (including custom "target_recording" field with the same signals)
```

Comment:
comment here should be something like this:

"path to Lhotse cuts manifest with noisy and corresponding clean speech signals (clean signals should be in "target_recording" field)"